### PR TITLE
Create BUG-REPORT.yml

### PR DESCRIPTION
**Pull request by Tieumsan**
Adding the BUG-REPORT issue template to help us provide a good developer experience while contributing to SORA.

### CODE REVIEW

**Comment by Tieumsan** [comment]
This is a form. cf. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

@bgodlin

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  value: |
+    Thank you for helping us improving SORA!
+    
+  - type: textarea
+  id: preconditions
+  attributes:
+    label: Preconditions
+    description: What are the preconditions to reproduce the bug ?
+  validations:
+    required: true
```
Comment (Ada-Agava) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:18]: I believe that precondition shouldn't be the mandatory field. We can have bugs without preconditions, for example "swap returns error to all users"

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  id: expected-behavior
+  attributes:
+    label: Expected behavior
+    description: A clear and concise description of what you expected to happen.
+  validations:
+    required: true
+
+# Do we inform about OS and version in dropdown or single-line text ? (overhead to ensure all OS/versions are represented)
+  - type: input
+  id: os
```
Comment (Ada-Agava) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:50]: Maybe we need the device type and model here also?

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    description: What operating system version of the OS are you using when encountering the bug ?
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context
+      description: Add any other context about the problem here. If applicable, add screenshots to help explain your problem.
+
```
Comment (Ada-Agava) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:62]: please let's add the environment (dev/test/prod) - mandatory
and the priority - not mandatory
fields here

**Pull request merged**